scenarios/bigquery.py

```diff
@@ -27,6 +28,8 @@
 
 import yaml
 
+import influxdb
```

Comment:
Scenarios should not import anything.

Perhaps what you really want to do is: a) move this into metrics/bigquery.py and b) change this to an execute scenario?